commons/pom.xml

```diff
@@ -158,7 +157,6 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>jcl-over-slf4j</artifactId>
-            <scope>test</scope>
```

Comment:
Log implementations should be compile only on final .jars/.wars

These are here to provide a log implementation for the tests and the scope test should not be removed